### DISCUSSION TITLE
Defining a irregular shaped domain to fit a tesr file

### DISCUSSION THREAD

**Discussion by inandi2** [neperfepx/neper#840] [General]
Hi,
I hope you are doing great. I am currently working on to mesh a tesr file of irregular shape (near cylindrical). The approach I am currently following is to fit the tesr into a cylindrical domain and trim everything outside of it. I was wondering if there is a way to define a domain exactly fitting the shape of the tesr. Thank you.

**Comment by rquey** [comment]
If the domain is convex, use the various possibilities of `-domain` to represent it as well as possible.  If it is not convex, you can also use `-transform cut`.
